src/compiler/binder.ts

```diff
@@ -210,6 +216,24 @@ namespace ts {
             return propAccess.name.text;
         }
 
+        /**
+         * Returns the first invocation of an identifier 'require' with one argument,
+         * or undefined if that none exist.
+        */
+        function findCommonJSRequireCalls(node: Node): Node {
+            if (node.kind === SyntaxKind.CallExpression) {
+                let call = <CallExpression>node;
+                if(call.expression.kind === SyntaxKind.Identifier &&
+                    (<Identifier>call.expression).text === 'require' &&
+                    call.arguments.length === 1) {
+
+                    return node;
+                }
+            }
+
+            return forEachChild(node, findCommonJSRequireCalls);

```

Comment:
Can you give an example of the `require` appearing at the child position? Do we need a test for that case?